nilearn/input_data/tests/test_nifti_labels_masker.py

```diff
@@ -368,3 +367,24 @@ def test_nifti_labels_masker_with_mask():
         masked_labels, resampling_target=None, mask_img=mask_img)
     masked_signals = masked_masker.fit().transform(fmri_img)
     assert np.allclose(signals, masked_signals)
+
+def test_3d_images():
+    # Test that the NiftiLabelsMasker works with 3D images
+    affine = np.diag((4, 4, 4, 1))
```

Comment:
Is there a reason you used this affine instead of the more classic `np.eye(4)`?

Reply:
@NicolasGensollen I used one of the tests already provided with the same affine then i found out that the affine by default is enough for testing so i changed it